spec/services/solr_mapper_spec.rb

```diff
@@ -54,7 +63,8 @@
             rights_uris_sim: ['https://creativecommons.org/licenses/by/4.0/'],
             affiliation_names_sim: ['My institution', 'B. Parent Organization', 'My contributor institution'],
             variables_tsim: ['variable 1', 'variable 2'],
-            temporal_isim: [2022]
+            temporal_isim: [2022],
+            available_at_struct_ss: [{ identifier: '10.1234/5678', provider: 'DataCite' }, { identifier: 'redivis-123', provider: 'Redivis' }]
```

Comment:
The struct_ss field is just a string field (see creators_struct_ss for example).

Reply:
Also, to simplify/streamline the code, one suggestion is to add provider_identifier_map to the struct_fields method.  The solr field can be called provider_identifier_map_struct_ss (or whatever the name would be automatically with the struct fields method), so we can just fit this into that method.

Reply:
Fixed.